_overviews/scala3-book/scala-for-python-devs.md

```diff
@@ -0,0 +1,1041 @@
+    </tr>
+    <tr>
+      <td valign="top"><code>if x == 1:
+        <br>&nbsp; print("x is 1, as you can see:")
+        <br>&nbsp; print(x)</code>
+      </td>
+      <td valign="top"><code>if x == 1 then
+        <br>&nbsp; println("x is 1, as you can see:")
+        <br>&nbsp; println(x)</code>
+      </td>
+    </tr>
+
+    <tr>
+      <td colspan="2" class="table-desc-row">if/else-if/else</td>
+    </tr>
+    <tr>
+      <td valign="top"><code>if x &lt; 0:
+        <br>&nbsp; print("negative")
+        <br>elif x == 0:
+        <br>&nbsp; print("zero")
+        <br>else:
+        <br>&nbsp; print("positive")</code>
+      </td>
+      <td valign="top"><code>if x &lt; 0 then
+        <br>&nbsp; println("negative")
+        <br>else if x == 0
+        <br>&nbsp; println("zero")
+        <br>else
+        <br>&nbsp; println("positive")</code>
+      </td>
+    </tr>
+
+    <tr>
+      <td colspan="2" class="table-desc-row">Returning a value from <code>if</code></td>
+    </tr>
+    <tr>
+      <td valign="top"><code>min_val = a if a &lt; b else b</code></td>
+      <td valign="top"><code>val minValue = if a &lt; b then a else b</code></td>
+    </tr>
+
+    <tr>
+      <td colspan="2" class="table-desc-row"><code>if</code> as the body of a method</td>
+    </tr>
+    <tr>
+      <td valign="top"><code>def min(a, b):
+        <br>&nbsp; return a if a &lt; b else b</code>
+      </td>
+      <td valign="top"><code>def min(a: Int, b: Int): Int =
+        <br>&nbsp; if a &lt; b then a else b</code>
+      </td>
+    </tr>
+
+    <tr>
+      <td colspan="2" class="table-desc-row"><code>while</code> loops</td>
+    </tr>
+    <tr>
+      <td valign="top"><code>i = 1
+        <br>while i &lt; 3:
+        <br>&nbsp; print(i)
+        <br>&nbsp; i += 1</code>
+      </td>
+      <td valign="top"><code>var i = 1
+        <br>while i &lt; 3 do
+        <br>&nbsp; println(i)
+        <br>&nbsp; i += 1</code>
+      </td>
+    </tr>
+
+    <tr>
+      <td colspan="2" class="table-desc-row"><code>for</code> loop with range</td>
+    </tr>
+    <tr>
+      <td valign="top"><code>for i in range(0,3):
+        <br>&nbsp; print(i)</code>
+      </td>
+      <td valign="top"><code>// preferred
+        <br>for i &lt;- 0 until 3 do println(i)
+        <br>
+        <br>// also available
+        <br>for (i &lt;- 0 until 3) println(i)
+        <br>
+        <br>// multiline syntax
+        <br>for
+        <br>&nbsp; i &lt;- 0 until 3
+        <br>do
+        <br>&nbsp; println(i)</code>
+      </td>
+    </tr>
+
+    <tr>
+      <td colspan="2" class="table-desc-row"><code>for</code> loop with a list</td>
+    </tr>
+    <tr>
+      <td valign="top"><code>for i in ints: print(i)
+        <br>
+        <br>for i in ints:
+        <br>&nbsp; print(i)</code>
+      </td>
+      <td valign="top"><code>// preferred
+        <br>for i &lt;- ints do println(i)
+        <br>
+        <br>// also available
+        <br>for (i &lt;- ints) println(i)</code>
```

Comment:
I would drop this, we are not giving any examples for `if`, so I think we should stay with optional braces and avoid confusion.